src/tesk_core/filer.py

```diff
@@ -506,13 +627,16 @@ def main():
 
     for afile in data[args.transputtype]:
         logging.debug('Processing file: %s', afile['path'])
-        if process_file(args.transputtype, afile):
-            logging.error('Unable to process file, aborting')
-            return 1
+        process_file(args.transputtype, afile)
         logging.debug('Processed file: %s', afile['path'])
 
     return 0
 
 
 if __name__ == "__main__":
     sys.exit(main())
+    # loglevel = logging.INFO
+    # logConfig(loglevel)
+    # data = json.loads(open("/home/cibin/EBI/tesk-core/examples/inputS3.json").read())
+    # for afile in data["inputs"]:
+    #     process_file("inputs",afile)
```

Comment:
Why has this commented code made it to the PR?

Reply:
Removed